<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>AccountTreeWidget</class>
 <widget class="QWidget" name="AccountTreeWidget">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>400</width>
    <height>300</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <property name="locale">
   <locale language="English" country="UnitedKingdom"/>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout">
   <property name="spacing">
    <number>6</number>
   </property>
   <property name="leftMargin">
    <number>0</number>
   </property>
   <property name="topMargin">
    <number>0</number>
   </property>
   <property name="rightMargin">
    <number>0</number>
   </property>
   <property name="bottomMargin">
    <number>0</number>
   </property>
   <item>
    <layout class="QHBoxLayout" name="controlsHorizontalLayout">
     <item>
      <widget class="QToolButton" name="expandAllToolButton">
       <property name="text">
        <string>...</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QToolButton" name="collapseAllToolButton">
       <property name="text">
        <string>...</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QToolButton" name="addAccountGroupToolButton">
       <property name="text">
        <string>...</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QToolButton" name="addCashAccountToolButton">
       <property name="text">
        <string>...</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QToolButton" name="addSecurityAccountToolButton">
       <property name="text">
        <string>...</string>
       </property>
      </widget>
     </item>
     <item>
      <spacer name="horizontalSpacer">
       <property name="orientation">
        <enum>Qt::Horizontal</enum>
       </property>
       <property name="sizeHint" stdset="0">
        <size>
         <width>40</width>
         <height>20</height>
        </size>
       </property>
      </spacer>
     </item>
    </layout>
   </item>
   <item>
    <widget class="QTreeView" name="treeView">
     <property name="sizePolicy">
      <sizepolicy hsizetype="MinimumExpanding" vsizetype="Expanding">
       <horstretch>0</horstretch>
       <verstretch>0</verstretch>
      </sizepolicy>
     </property>
     <property name="sizeAdjustPolicy">
      <enum>QAbstractScrollArea::AdjustToContents</enum>
     </property>
     <attribute name="headerMinimumSectionSize">
      <number>35</number>
     </attribute>
     <attribute name="headerHighlightSections">
      <bool>true</bool>
     </attribute>
     <attribute name="headerStretchLastSection">
      <bool>false</bool>
     </attribute>
    </widget>
   </item>
  </layout>
  <action name="actionAdd_Account_Group">
   <property name="text">
    <string>Add Account Group</string>
   </property>
  </action>
  <action name="actionAdd_Security_Account">
   <property name="text">
    <string>Add Security Account</string>
   </property>
  </action>
  <action name="actionAdd_Cash_Account">
   <property name="text">
    <string>Add Cash Account</string>
   </property>
  </action>
  <action name="actionEdit">
   <property name="text">
    <string>Edit</string>
   </property>
  </action>
  <action name="actionDelete">
   <property name="text">
    <string>Delete</string>
   </property>
  </action>
  <action name="actionExpand_All">
   <property name="text">
    <string>Expand All</string>
   </property>
  </action>
  <action name="actionCollapse_All">
   <property name="text">
    <string>Collapse All</string>
   </property>
  </action>
  <action name="actionExpand_All_Below">
   <property name="text">
    <string>Expand All Below</string>
   </property>
  </action>
 </widget>
 <resources/>
 <connections/>
</ui>
